module/VuFind/src/VuFind/Controller/AbstractBase.php

```diff
@@ -51,11 +51,12 @@ class AbstractBase extends AbstractActionController
 {
     /**
      * Permission that must be granted to access this module (false for no
-     * restriction)
+     * restriction, null to use configured default (which is usually the same
+     * as false)).
      *
      * @var string|bool
```

Comment:
string|bool|null? Or is null considered typeless?

Reply:
I believe that null is considered typeless; I rarely see doc comments that explicitly reference null.